encryption_works.md

```diff
@@ -98,25 +98,58 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
 
 ## Anonymize Your Location with Tor
 
-Tor is a software service that allows you to use the Internet while concealing your IP address, which is, in general, a fairly accurate representation of your location. The [Tor](https://www.torproject.org/) network is made up of over 3,600 volunteer servers called nodes. When someone uses the Tor network to visit a website their connection gets bounced through three of these nodes (called a circuit) before finally exiting into the normal Internet. Anyone intercepting traffic will think your location is the final node which your traffic exits from.
+While your web habits can often be tracked when browsing the Internet, it's still possible to use the web and avoid many types of mass surveillance. The easiest way is to use the Tor Browser, a web browser like Firefox or Safari, but it allows you to browse the web while masking your IP address, which is generally an accurate representation of your approximate location. 
+
+The software is free and open-source, meaning that it comes with a high degree of trust, and is an adapted version of the Firefox browser.
+
+![The Tor Browser](https://raw.githubusercontent.com/tommycollison/encryption-works/master/images/tor_FPF.png)
+
+The Tor network is made up of over 6,000 volunteer servers, called nodes. When you use Tor, your connection is routed through three of these nodes — called a circuit – before exiting onto the normal Internet. Anyone intercepting Internet traffic will think your location is the final node which your traffic exits from.
+
+![To obscure your location, Tor routes your connection through three intermediaries.](/images/tor_relays.png)
+
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). Once you download and install it, you'll see that it works like any other web browser but is a little bit slower, since your traffic is being routed through the Tor network to provide you with anonymity.
+
+When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
```

Comment:
Some clarification about bridges.

Bridges are meant to provide a connection to a first-hop Tor relay for users who are on a censored Internet connection. They work when the censor only blocks connections based on their IP address. Since the list of Tor relays is publicly available, it is easy for them to use it to build a blocklist that includes all Tor relays, thus preventing a user from making any connection to the Tor network.

Bridges are relays that do not have a publicly published IP address. Their IPs are meant to be communicated out-of-band on a channel that is less likely to be blocked by the censor (e.g. over email or in person). Unfortunately even this is not immune to surveillance - thanks to Snowden, we know that NSA had XKeyscore rules that extracted Tor bridge addresses from the emails sent by the Tor Project (see "Not only metadata" in the [original article](http://daserste.ndr.de/panorama/aktuell/nsa230_page-3.html)).

So this sentence is technically correct - using a bridge might make it less obvious that you are using Tor to a *dumb* eavesdropper that only profiles you based on the IP addresses you connect to. However, these days most adversaries are not that dumb (and Snowden's adversaries never were).

You can think of bridges by themselves as version 1.0 of Tor's censorship resistance defenses. As censors got smarter (partially motivated by the development of censorship circumvention techniques such as bridges), they started using deep packet inspection (DPI) to profile user's traffic flows. Even if they used a bridge, by an eavesdropper could determine whether or not they were using Tor by analyzing various characteristics of their traffic that were "Tor-like".

Tor's response to this has been [pluggable transports](https://www.torproject.org/docs/bridges.html.en#PluggableTransports), which modify the Tor traffic in various ways to make it look less like Tor traffic and more like other traffic (say, a Skype video call, or some random TLS connection to a website, depending on the specific transport being used). For technical reasons, currently pluggable transports are used in conjunction with bridges - either a set of bridges included with the Tor Browser (if you're more interested in just hiding that fact that you're using Tor) or out-of-band obtained bridges (if your connection is still censored based on IP address).

So if Ed wanted to hide the fact that he was using Tor, a bridge + pluggable transports might've helped.

The sentence as-is is correct, so you don't need to change it right now. I just wanted to make sure everybody was on the same page about the different properties of bridges.